Trimble.Modus.Components/Controls/TMSwitch/TMSwitch.xaml.cs

```diff
@@ -24,27 +19,37 @@ public partial class TMSwitch : ContentView
     public static readonly BindableProperty LabelTextProperty =
       BindableProperty.Create(nameof(Text), typeof(string), typeof(TMSwitch), "", propertyChanged: OnLabelTextChanged);
     public static readonly BindableProperty TextColorProperty =
-        BindableProperty.Create(nameof(TextColor), typeof(Color), typeof(TMSwitch), Colors.Black);
+        BindableProperty.Create(nameof(TextColor), typeof(Color), typeof(TMSwitch), Colors.Transparent, propertyChanged: OnTextColorChanged);
+
     public new static readonly BindableProperty BackgroundColorProperty =
-        BindableProperty.Create(nameof(BackgroundColor), typeof(Color), typeof(TMSwitch), Colors.Black);
+        BindableProperty.Create(nameof(BackgroundColor), typeof(Color), typeof(TMSwitch), Colors.Transparent, propertyChanged: OnBackgroundColorChanged);
+
+    public static readonly BindableProperty ThumbColorProperty =
+       BindableProperty.Create(nameof(ThumbColor), typeof(Color), typeof(TMSwitch), Colors.Transparent, propertyChanged: OnThumbColorChanged);
 
     public string Text
     {
         get => (string)GetValue(LabelTextProperty);
         set => SetValue(LabelTextProperty, value);
     }
-    public Color TextColor
+    internal Color TextColor
     {
         get => (Color)GetValue(TextColorProperty);
         set => SetValue(TextColorProperty, value);
     }
 
-    public new Color BackgroundColor
+    internal new Color BackgroundColor
     {
         get => (Color)GetValue(BackgroundColorProperty);
         set => SetValue(BackgroundColorProperty, value);
     }
 
+    internal Color ThumbColor
```

Comment:
Shouldn't this be public as well?

Reply:
Via styling, we pass these colors. So changed inline styling as internal as followed in other controls